content/1.7/back-office/international/locations/countries/add-edit-country.md

```diff
@@ -0,0 +1,74 @@
+
+ - **Address format** - Firstly a box is displayed with the following default inputs: firstname lastname, company, address1, address2, city, State:name postcode, Country:name, phone. On the right a helptext is displayed: "Some countries require different elements than others. Click on the button below to get the valid default address format for this country."
+Under the help text, the 5 following helper links are displayed and can be selected.  You can click on the various helper links on the side of the text-field in order to add more fields to the address format. Adding the same elements in the address format, for example "lastname" multiple times, when trying to "save" will cause errors:
+ - "This property does not exist inside class, or it is forbidden.: Address: id_phonecustomer"
+ - "This class name does not exist.: PhoneCustomer"
+ - "This key has already been used.: Customer:lastname"
+Editing or adding the "Address format" incorrectly when trying to "save" will also cause the following errors:
+ - "This property does not exist inside class, or it is forbidden.: Address:"
+ - "This name is not allowed.:"
+If the address format field if some of the needed fields are empty, when trying to "save" will show the following errors:
+ - "The firstname field (in tab Address) is required."
+ - "The lastname field (in tab Address) is required."
+ - "The address1 field (in tab Address) is required."
+ - "The city field (in tab Address) is required."
+ - "The Country:name field (in tab Country) is required."
+ - "Address format invalid".
+
+     - **Customer** -  the following parameters can be added to the address format: lastname, firstname, birthday, email, website, company, siret, reset_password_token, htmlFields. 
+      - **Warehouse** - the following parameters can be added to the address format: reference, name, management_type, htmlFields
+     - **Country** - the following parameters can be added to the address format: iso_code, name, zip_code_format, htmlFields
+     - **State** - the following parameters can be added to the address format: iso_code, name, htmlFields 
+     - **Address** - ithe following parameters can be added to the address format: company, lastname, firstname, address1, address2, postcode, city, other, phone, phone_mobile, vat_number, dni, htmlFields.
+When a parameter is already existing, when trying to save it, the error shows: "This key has already been used.: phone"
+     
+ Below there are 4 buttons with different actions:
+ 
+     - **Use the last registered format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel".
```

Comment:
```suggestion
     - **Use the last registered format** - When this button is pressed, the following modal is displayed: "Are you sure you want to restore the default address format for this country?", the options are: "ok", "cancel". A tooltip is displayed on button hovered: "This will restore your last registered address format"
```